Util/Xamarin.Kotlin.BindingSupport/native/src/main/kotlin/xamarin/binding/kotlin/bindingsupport/Processor.kt

```diff
@@ -196,13 +203,13 @@ class Processor(xmlFile: File, jarFiles: List<File>, outputFile: File?) {
 
         // remove the members that are not meant to be used
         processMembers(xclass, "constructor") {
-            shouldRemoveMember(it, jclass.declaredConstructors)
+            shouldRemoveMember(it, jclass.declaredConstructors + jclass.constructors)
```

Comment:
This is still a required change because several members are not found for some reason. It works now, so...